### PR TITLE
Updated Changelog for 9.0.2 release

### PR DESCRIPTION
**Pull request by claco**

### CODE REVIEW

File: Changelog.md [hunk 1]
```diff
@@ -1,9 +1,26 @@
 # Changelog
 
+## 9.0.2 - 2014-11-05
+
+- Increase delay and retries for lxc cache download [#449]
+- Updated cinder.conf to allow for AZ setup [#458]
+- Use {{ ansible_fqdn }} for service checks [#452]
+- Allow for more galera/mysql tuning [#410,#429]
+- Increase ssh timeout in ansible.cfg [#358]
+- Updated pip installation/wheell building process
```
Comment (sigmavirus24) [inline Changelog.md]: "Wheel" has one "l"

File: Changelog.md [hunk 2]
```diff
@@ -1,9 +1,26 @@
...
+- Created lxc-system-manage script for common operational tasks [#434]
+- Added missing comma which caused Kibana dashboard to not load
+- Updated rpc_release/maas_repo_version versions [#370]
+- Ensure that rsyslog state files are unique [#205]
+- Changed release version to match the branch [#421]
+- Changed maas_notification_plan to npManaged [#402]
+- Added galera alarms [#403]
+- Resolved issue with neutron ha failover cron clobbering other crons [#383,#378]
```
Comment (sigmavirus24) [inline Changelog.md]: `"ha".upcase`?

**Comment by sigmavirus24** [comment]
:+1:

**Comment by stevelle** [comment]
LGTM

**Pull request merged**